src/stylesheets/elements/_inputs.scss

```diff
@@ -77,8 +77,19 @@ select {
   label {
     margin-top: 0;
   }
+
+  .usa-input-inline {
+    border: $input-border-width solid $color-gray;
+    width: inherit;
+  }
+
+  .usa-input-inline-error {
+    border: 3px solid $color-secondary-dark;
```

Comment:
Hi @jasonkaedingICE, thanks for your question. I'm going to cc @juliaelman from the core team to help you with this as I have moved to a different project.